src/AaveV2FreezeReserves_20230619/V2-to-V3-MIGRATION-NEXT-STEPS.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Chaos Labs V2 to V3 Migration Next Steps
+author: Chaos Labs (@Chaos Labs)
+shortDescription: Chaos Labs V2 to V3 Migration Next Steps
+discussions: https://governance.aave.com/t/arfc-chaos-labs-v2-to-v3-migration-next-steps/13701
+created: 2023-06-19
```

Comment:
```suggestion
```
These field is unused and it's not quite clear what it even sais.
`created` meaning creation of the md file, or the proposal? The first is meaningless, the second can be fetched on-chain. Better remove it.